engine/runtime/src/main/java/org/enso/compiler/context/CompilerContext.java

```diff
@@ -57,9 +60,20 @@ public interface CompilerContext {
   void truffleRunCodegen(Source source, ModuleScope scope, CompilerConfig config, IR.Module ir);
 
   ExpressionNode truffleRunInline(
-      Source source, InlineContext scope, CompilerConfig config, IR.Expression ir);
+      Source source, LocalScope localScope, Module module, CompilerConfig config, IR.Expression ir);
 
   // module related
+
+  void runStubsGenerator(Module module);
```

Comment:
we should probably start adding some documentation here

Reply:
There is little point of documenting individual methods - their presence is random, driven by needs of the overall plan. I've stated it in the class javadoc: [10bd526](https://github.com/enso-org/enso/pull/7666/commits/10bd526a0fb69a1f0eff2f401652c3b1aa1f1394)